.github/workflows/build.yml

```diff
@@ -0,0 +1,15 @@
+on:
+  pull_request_target:
```

Comment:
Enables safe builds for fork-pr workflows for contributors from outside the repository

Reply:
This is effectively saying when a pull request is opened or pushed to run the build job in a way that is compatible with forks?

Reply:
Yep! Specifically the [pull_request_target](https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#pull_request_target) configuration uses the target branch GitHub Actions configuration. That way a drive-by user cannot create a PR that modifies the workflow actions to expose secrets in the build.